README.md

```diff
@@ -153,13 +153,13 @@ Update `data_dir` configuration in config file to a different directory.
 or using a simple shell command to modify the config file:
 
 ```shell
-ex -sc '%s/"data_dir": "default"/"data_dir": "node2"/|x' nodes/node2.json
+ex -sc '%s/"data_dir": "default"/"data_dir": "node2"/|x' nodes/node2.toml
```

Comment:
The replace command should be changed also.

Reply:
Removed the `ex` as it's overshooting for a single line change.